docs/reference/api/platform/Logger.md

```diff
@@ -0,0 +1,164 @@
+## Logger
+
+Mbed OS provides a set of API's that you can use to output different log level messages to STDIO at runtime. mbed_logger.h declares these functions, which are available only in debug builds.
+API's in logging module are printf-style API's which take module name and format string followed by arguments.
+
+By default all messages till LOG_LEVEL_DEBUG are enabled. If you want to enable Trace and Info level messages, then set `MBED_CONF_MAX_LOG_LEVEL` accordingly.
```

Comment:
Is LOG_LEVEL_DEBUG a global setting? Where is it configured?

Reply:
MBED_CONF_MAX_LOG_LEVEL - MBED_CONF are all config setting globals.